config/routes.rb

```diff
@@ -0,0 +1,20 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
+  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  get "up" => "rails/health#show", as: :rails_health_check
```

Comment:
```suggestion
  get "healthcheckz" => "rails/health#show", as: :rails_health_check
```

For some context this is what all of the apps use for liveness checks by the load balancer